packages/components/rollup.config.mjs

```diff
@@ -92,40 +120,45 @@ export default {
           src: "src/Combobox/components/ComboboxContent/ComboboxContentList/ComboboxContent.css.d.ts",
           dest: "dist/Combobox/components/ComboboxContent/ComboboxContentList",
         },
+        {
```

Comment:
publint prefers if an mts types file is present for mjs/esm builds, so that's why it gets copied and pasted to a new extension.